debian/rules

```diff
@@ -23,8 +23,8 @@ export PYBUILD_INSTALL_ARGS=--install-script /usr/sbin
 %:
 	dh $@ --with python3 --buildsystem pybuild
 
-override_dh_instalsystemd:
-	dh_install_systemd -ppdudaemon --name=pdudaemon --no-restart-on-upgrade share/pdudaemon.service
+override_dh_installsystemd:
+	dh_installsystemd -ppdudaemon --name=pdudaemon --no-restart-on-upgrade pdudaemon.service
```

Comment:
but the service file is in `share/pdudaemon.service` in the source tree right?

Reply:
Yes, but `dh_installsystemd` needs a name of the unit (already installed in the correct place), not a path.

Reply:
ah i see and i didn't notice the snippet name also got changed; tbh if that snippet never ran we may as well drop it entirely. Fwiw for future if you could make your commit message a bit more clear then it saves me for asking :)